eth/filters/filter_system.go

```diff
@@ -399,6 +440,10 @@ func (es *EventSystem) lightFilterNewHead(newHeader *types.Header, callBack func
 	}
 }
 
+func (es *EventSystem) lightFilterNewSideHead(header *types.Header, callBack func(*types.Header, bool)) {
+	callBack(header, true)
```

Comment:
This is noteworthy; logs from side chain blocks will always be marked `Removed: true`. `newHeads` handling of this uses `Removed` to indicate the same characteristic, but in that context is expected to _have been_ canonical, and is now _removed_ to the side. Although the reuse of this field (and vocabulary) is a little awkward, I think it's the best we can do without modifying the existing API data type.